frontend/pages/ManageControlsPage/OSSettings/cards/Certificates/components/AddCertificateModal/AddCertificateModal.tsx

```
@@ -145,8 +148,13 @@ const AddCertModal = ({
       const sanReason = getErrorReason(e, {
         nameEquals: "subject_alternative_name",
       });
+      const nameConflict = getErrorReason(e, {
+        reasonIncludes: "already exists",
+      });
       if (sanReason) {
         setServerErrors({ subjectAlternativeName: sanReason });
+      } else if (nameConflict) {
+        setServerErrors({ name: "Name is already used by another certificate." });
```

ESLint

[error] Replace `·name:·"Name·is·already·used·by·another·certificate."` with `⏎··········name:·"Name·is·already·used·by·another·certificate.",⏎·······`

(prettier/prettier)